soc/nordic/common/uicr/gen_uicr/prj.conf

```diff
@@ -1 +1,6 @@
-# nothing here
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# Enable UICR generator options - this is automatically set when building
+# the gen_uicr image and enables the UICR generator configuration menu
+CONFIG_IS_GEN_UICR_IMAGE=y
```

Comment:
Does this prj.conf file get overridden when we define a conf file at sysbuild/uicr.conf in a sample directory, or does that work as an overlay?